src/fsharp/FSComp.txt

```diff
@@ -80,7 +80,7 @@ tastTypeHasAssemblyCodeRepresentation,"The type '%s' has an inline assembly code
 255,tastUnexpectedByRef,"Unexpected use of a byref-typed variable"
 256,tastValueMustBeLocalAndMutable,"A value must be mutable in order to mutate the contents or take the address of a value type, e.g. 'let mutable x = ...'"
 257,tastInvalidMutationOfConstant,"Invalid mutation of a constant expression. Consider copying the expression to a mutable local, e.g. 'let mutable x = ...'."
-tastValueHasBeenCopied,"The value has been copied to ensure the original is not mutated by this operation"
+tastValueHasBeenCopied,"The value has been copied to ensure the original is not mutated by this operation or because the copy is implicit when returning a struct from a member and another member is then accessed"
```

Comment:
I think the message now is very complete, but could be shortened ... perhaps ... ,"The value has been copied to ensure the original is not mutated by this operation or copy is implicit when returning a struct from a member"